fs/dafsserver/dafsserver.cpp

```diff
@@ -2789,13 +2789,13 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
         StructArrayOf<OpenFileInfo> openFiles;
         Owned<IDirectoryIterator> opendir;
         unsigned            lasttick, lastInactiveTick;
-        atomic_t            &globallasttick;
+        std::atomic<unsigned> &globallasttick;
```

Comment:
Not new, but this feels more like a volatile than an atomic?

Reply:
I think atomic is correct.  volatile was really meant for memory mapped io devices, where you had to guarantee that data was  re-read.